### ISSUE TITLE
The API for font building is inadequate

### ISSUE THREAD

**Issue by GrimMaple** [mellinoe/ImGui.NET#492]
I've ran into #127 myself. Though the proposed solution does _work_, I find it inadequate from the API viewpoint

**Comment by zaafar** [comment]
yeah i know, in another project, I have created a wrapper API on top of imgui.net to bring QOL into font replacement activity. following is the example of how I am using that wrapper API.

https://github.com/zaafar/ClickableTransparentOverlay/blob/master/Examples/SingleThreadedOverlayWithCoroutines/SampleOverlay.cs#L114

maybe you can do the same in your project?

**Comment by GrimMaple** [comment]
Well, making wrappers is OK(~ish). This is just an opportunity to improve this project. But unfortunately, I don't know where to begin with this. Maybe I could spend some time on it if you know how this can be approached

**Comment by zaafar** [comment]
Another thing is that Imgui folks are thinking about changing it so whatever change imgui.net folks will do would become invalid in the future...
